.github/workflows/update.yaml

```diff
@@ -0,0 +1,56 @@
+name: Meshery Istio Pattern Components Generator
+on:
+  push:
+    branches:
+      - 'master'
+  schedule:
+  - cron: "0 0 * * *"
+jobs:
+  GenerateComponents:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@master
+    - name: Setup Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.17
+    - name: Build jsonschema util
+      run: |
+          wd=$(pwd)
+          mkdir -p ~/.meshery/bin
+          cd ~/.meshery/bin
+          curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
+          export NVM_DIR="$HOME/.nvm"
+          [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+          nvm install v14.15.3
+          nvm install-latest-npm
+          git clone https://github.com/layer5io/kubeopenapi-jsonschema util
```

Comment:
Is this temporary until switching over to running an instance of the adapter itself?

Reply:
It still would need kubeopenapi-jsonschema. We are using this to build the binary

Reply:
Makes much sense that the binary will still be around. What I'm asking is why we're still scripting around the binary and not calling the binary from what will be a Golang-wrapped approach consistently used across every Adapter and Server.

Reply:
To be clear, components will still be generated in a workflow and persisted in this repo. Is this work completed already?

Reply:
It seems that the new workflow covers this now, is that right?